R/plumber-shared.R

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Shared method between data-storage and plumber API to sign parameters (in telemetry) and perform validation (in plumber)